src/api/modules/workspace/create.ts

```diff
@@ -103,6 +103,12 @@ export const workspaceCreator: WorkspaceCreator = async ({
 
     return 'created'
   } catch (err) {
+    if (err.response?.data.code === 'ArgumentNull') {
+      throw createFlowIssueError(
```

Comment:
@VictoriaMolgado helped me about this new error flow to the user, what do you guys think about it ?

Reply:
Should we add some colors highlighting the code parts, or '' enclosing them? In this part for example:
```
Run vtex workspace create [WORKSPACENAME]
```
Should we print?
```
Run 'vtex workspace create [WORKSPACENAME]'
```
Should we add colors in it?

Reply:
Good ideia
![](https://i.imgur.com/yJ4Xilx.png)

Reply:
Nicee, what do you think abouot the last one?
`for example vtex workspace create johndoe01`
Should we color it too?

Reply:
Or maybe just enclose it in ''

Reply:
Option 1,2 or 3 ?
![](https://i.imgur.com/Qr8kUG1.png)

Reply:
I think those should be VTEX pink, since now we have defined colors for our CLI, you can see it in [here](https://www.figma.com/file/GpNy0htTpJ7uvw7WKqGdLH/Toolbelt-DX-review?node-id=373%3A89) and they are on `ColorifyConstants` 😄

Reply:
Use the green color for now :) 
That's because we want to use the vtex pink only for commands. Checkout this proto-guide: https://www.notion.so/Toolbelt-Styleguide-1943920c5cb84d4a9e22a501c43beab2#0f28a0efe490471f9aca3424d3a01936

Is it possible to associate a class to the line to change it easily if it makes sense in the future?